ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -26,10 +34,18 @@
                  :subscription provider-id native-id body content-type headers)
         _ (lt-validation/validate-launchpad-token request-context)
         _ (api-core/verify-provider-exists request-context provider-id)
-        _ (acl/verify-ingest-management-permission
-            request-context :update :provider-object provider-id)
-        _ (acl/verify-subscription-management-permission
-            request-context :update :provider-object provider-id)
+        subscription-user (get (json/decode (:metadata concept)) "SubscriberId")
+        token-user (api-core/get-user-id request-context headers)
+        bypass-acls (subscription-user-matches-token? concept request-context headers)
```

Comment:
I'm not sure about the logic.  When you UPDATE a subscription, I can understand that when the token is the same as the subscriber-id, you don't have to check the permission. But when you INGEST a brand new subscription, you will still have to check permission right? regardless if the subscriber himself is submitting the request or the operator is submitting the request on his behalf.

Reply:
If I understand you correctly, no, the idea was to allow people without this ACL (from Earthdata Search) to create their own. So I'm not sure I see what you mean.

Reply:
Even if Earthdata Search requires no ACL on subscription creation, what about the clients using CMR without going through Earthdata Search?  Do we want to allow anyone to create subscriptions without any ACLs?

Reply:
yes, anyone can create their own subscriptions. I tried to suggest a new interface just for earthdata search, or other limitations, they are not needed by the customer. So we are going to just log them for now. Several developers have suggested that we limit the number that a user can create to something like 100, that may happen in the future. You know what, I'll write a ticket to that effect and see what our ERB says.

Reply:
https://bugs.earthdata.nasa.gov/browse/CMR-6928 - I'm going to resolve this comment.